lib/zendesk_apps_tools/api_connection.rb

```diff
@@ -1,11 +1,23 @@
 module ZendeskAppsTools
   module APIConnection
-    FULL_URL     = /https?:\/\//
+    SUBDOMAIN    = /\A[a-z0-9][a-z0-9\-]{1,}[a-z0-9]\z/
+    FULL_URL     = Regexp::new(/https?:\/\//.source + SUBDOMAIN.source + /\.zendesk\.com/.source)
     URL_TEMPLATE = 'https://%s.zendesk.com/'
 
+    # Regular Expression for URI::MailTo::EMAIL_REGEXP in Ruby 2.4.0
+    EMAIL_REGEX  = /\A[a-zA-Z0-9.!\#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*\z/
```

Comment:
Maybe try `URI::MailTo::EMAIL_REGEXP || /\A[a-zA-Z0-9.!\#$%&'*+\/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*\z/`?

Reply:
Unfortunately the current built failed. 

Our Previous built, however, with URI::MailTo::EMAIL_REGEXP explicitly declared passed.

Reply:
Hey @Mehul-Porwal, I've decided to follow your advice in person and went with 

`EMAIL_REGEX  = URI::MailTo.constants.include?(:EMAIL_REGEXP) ? URI::MailTo::EMAIL_REGEXP : URI::MailTo::MAILTO_REGEXP`

<details>
<summary>Travis seems happy</summary>
<img width="1074" alt="screen shot 2018-11-19 at 5 55 27 pm" src="https://user-images.githubusercontent.com/17760485/48690891-7aeb0300-ec24-11e8-8990-e799bac7ebe1.png">
</details>